util/Migrator/DbScripts/2023-09-14_00_Remove_WebAuthn_For_Free_Users.sql

```diff
@@ -0,0 +1,39 @@
+-- Assumptions:
+-- When a 2FA method is disabled, it is removed from the TwoFactorProviders array
+
+-- Problem statement:
+-- We have users who currently do not have any 2FA method, with the only one being
+-- WebAuthn, which is effectively disabled by a server-side permission check for Premium status.
+-- With WebAuthn being made free, we want to avoid these users suddenly being forced
+-- to provide 2FA using a key that they haven't used in a long time, by deleting that key from their TwoFactorProviders.
+declare @TwoFactorByUser TABLE
+(
+    Id UNIQUEIDENTIFIER,
+    Email NVARCHAR(256),
+    TwoFactorType INT
+)
+
+INSERT INTO @TwoFactorByUser
+SELECT  u.Id, 
+        u.Email,
```

Comment:
⚠️ This is unused.

Reply:
Removed.  I had it on there to see what accounts I was getting back in testing, but not necessary any more.